examples/polar_pfm/verification/run.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+for var in upsilon theta
+do
+  for T in `seq 392 10 532`
```

Comment:
Does this file need to be checked in?

Reply:
It helps generating the data for the for the plots in the jupyter notebook. I could also have the notebook run magpie though

Reply:
That's good. Just making sure.

Reply:
I reworked the jupyter notebook to run the verification cases and generate all movies and plots